### DISCUSSION TITLE
Cosmology papers infodump

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#240] [Astrophysics and Physics]
Figured I might dedicated a thread to new papers of non major importance and I'll edit it instead of making new comments in order to not spam this forum too much (since new comments but not edits makes a post go up in the daily list)

So apparently (via the indirect proxy of light isotopes) there is a detection of liquid water 500 millions years earlier than previously thought. 
This should allow to rule out many candidate explanations for the extreme problem of the faint young the paradox 
https://en.m.wikipedia.org/wiki/Faint_young_Sun_paradox
Candidates are:
1) The expanding earth hypothesis (including the decompressing variant) 
2) The idea that the sun size is static or cyclic.( consistent with a universe that recycle matter, more precisely that recycle the lost energy of the tired light, e.g via neutrinos ) this explanation however is difficult to conciliate with the diversity of stars and the main sequence. (I still believe the universe is in equilibrium but this probably involve destructive and creative cycles of stars formation at a constant or self stabilizing cosmic rate)
3) the Gaia hypothesis or sylurian. (not very credible) 
4) the idea that contrary to the standard model, the universe expansion apply also to gravitationally bound objects aka the distance between de sun and earth was much shorter.
5) effects via modified theory of gravity or modified inertia.
6) greenhouse gases theories (mostly ruled out)
7) orbit change via foreign object interaction (e.g a foreign star or nebulae) 

Imo the gravitationally decompressing earth or expansion via serpentinization is the most likely explanation and the static or cyclic sun size comes in second place, local expansion in third. As for modified gravity or inertia I have no idea.

https://www.curtin.edu.au/news/media-release/fresh-findings-earliest-evidence-of-life-bringing-fresh-water-on-earth/

**Comment by LifeIsStrange** [comment]
@expEarth pinging you because you have studied expanding earth

**Comment by Francois-Zinserling** [comment]
May I add some speculation?

On the sun_paradox page there is a link to the 'Hadean' earth.
[Hadean](https://en.m.wikipedia.org/w/index.php?title=Hadean&diffonly=true#Cool_early_Earth)

During formation earth would have been hot and lava-like. The proposal is put forth that it cannot contain hydrogen, so no water can form (too hot and still too low gravity), but what if it could?

This idea came to me when I was studying an online 'Introduction to Astronomy' course many years ago with Prof. Ronen Plesser.  ... here goes ...

Known science first:
During the settling of the solar system, dust gravitates toward the sun, slow at first and then faster later as the sun's gravity increases. Once the sun has enough mass, fusion starts, and some of what is still falling in starts getting pushed back by radiation of the sun. 
[T-Tauri winds](https://en.m.wikipedia.org/wiki/T_Tauri_wind)

Then, by the time T-Tauri starts from the sun, the lighter gases - read: including lots of Hydrogen - is pushed back from the sun. This will be in the form of a thick shell around the sun, like a dust cloud in a strong wind, being slowly pushed back and the shell at first thickening, then much later gradually depleted until it is cleared, either fallen in to the sun, or collected by planets, or ejected out of the solar system from solar ray collisions. 

Speculation:
This T-Tauri shell may also have been the cause of 'faint early sun' more than the reason the earlier paradox article refers to. But that's another point.

Back to earth. During the infall of dust toward the sun, earth would have collected mass including at some point $O_2$, once it had enough gravity to retain it. Even though earth is 'hot', it is no longer a giant fireball, so oxygen remains mostly in gaseous form.

When this T-Tauri shell has been pushed back to the earth's orbital distance, earth will have maybe a few thousand (million?) years of hydrogen rich orbit. Earth will also have cooled off a bit more by now, and also gained some mass to have increased gravity. The T-Tauri shell itself will also have assisted with cooling earth's atmosphere.
Then with the more massive earth, already oxygen rich atmospere, hydrogen rich shell in the earth orbit, and earth with still some hot pockets from original forming, the recipe is complete to add water to earth. Lots of it. It was probably raining for a very very long time. No, not for 40 days, a few million years maybe.

To add to the speculation; Let's say the shell made it to Jupiter and Saturn.
By the rules of planet formation (from infalling dust) far-out planets like Jupiter and Saturn should more likely have been small and rocky, yet there they are large and gaseous .. If the T-Tauri pushed the shell to the orbits of Jupiter and Saturn, they would have had much longer time in the disk and accumulated a lot more of it. More hydrogen than anything else and since they were cooler for longer, gases were easily retained.
/end of hand-waving speculation

**Comment by ExpEarth** [comment]
The faint young sun paradox is not one I have studied much.  While I am sure the physics of the argument is sound, there is always the possibility of something having been missed.  The very creative speculation by @Francois-Zinserling gives a hint of the many possible scenarios that could be in play.  Are astrophysicists being a little too over-confident on this one?

Still, supposing the sun really was much fainter when the Earth began, I would go with the expanding Earth explanation (no surprise there!)  There are two aspects of this.  The Wiki article mentions the recent evidence by Křížek et al that the orbits of planets around the sun and of moons around their planets that I mentioned in a another thread: (https://arxiv.org/abs/2201.05311), (https://doi.org/10.1134%2FS0202289322020086).  Vavrycuk also wrote on this.  This idea is usually expressed as a local Hubble expansion running parallel with the cosmic expansion.  I have argued that it is only the local expansion that is happening, by virtue of the Hubble luminosity.  Basically, all objects are emitting energy in relation to their gravitational potential energy, _U_.  This energy causes repulsive forces within systems, leading to a local expansion.  On the cosmic scale, however, this force perfectly balances with gravity and so there is no cosmic expansion.

The second aspect relates to the direct heating effect of the Hubble luminosity on the primitive Earth.  The Earth would be smaller then and so the heating effect would be relatively greater than today.  On this topic, you may have seen all the recent evidence for planets and moons having more internal heat than can be explained.  Saturn recently came up again: [https://phys.org/news/2024-06-scientists-massive-energy-imbalance-saturn.html?utm_source=nwletter&utm_medium=email&utm_campaign=daily-nwletter](url).

Note added: This is a complex topic and I just scratched the surface of it.  If the Earth was originally only 2/3 the size of present, then the surface area would only have been 4/9 of present, less than half.  The atmosphere would thus have been doubled in thickness, other factors being the same.  The surface gravity would also have been more than double.  This means that this thicker Earth's atmosphere would  have been held twice as strongly.  The greenhouse effects would have been enhanced.

Here is an interesting paper by Kragh on the early days of the static universe and how it connected up with the expanding Earth model:
[https://arxiv.org/pdf/1507.08040](url)

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>the Hubble luminosity. Basically, all objects are emitting energy in relation to their gravitational potential energy, $U$.

Our main reference is about the Gravitational [Self Energy Field](http://www.alternatievewiskunde.nl/cosmology/nasa.htm), as has been mentioned in [another reply](https://github.com/orgs/a-cosmology-group/discussions/228#discussioncomment-9797503). Near the end of that page you will find the proper equations. What you call _gravitational potential energy_ should be the same as our self energy. But [your formula](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/HubbleRedshift.pdf) is missing a factor $2$ in the denominator. [Relativity Reexamined](http://www.alternatievewiskunde.nl/einstein/relativity-reexamined_compress.pdf) is a reference confirming this. Let $G=$ gravitational constant, $M=$ mass of the object, $R=$ radius of the object, then:

$$
U \equiv - \frac{GM^2}{R} \quad (2) \quad \mbox{must be} \quad U \equiv - \frac{GM^2}{2R} \quad (7.13)
$$

Furthermore, let $c=$ lightspeed, $r=$ radius. Then we can rewrite Newton's law of gravitation as follows (see main reference).

$$
F = GM\left[1-GM/(2 c^2 R)\right]/r^2 + G^2 M^2/(2c^2)/r^3
$$

The term $\left[1-GM/(2 c^2 R)\right]$ is lowering the "normal" gravitational force. This could be in concordance with
>This energy causes repulsive forces within systems, leading to a local expansion.

The term with $r^3$ causes a perihelion shift. However, as [discussed](https://github.com/orgs/a-cosmology-group/discussions/214#discussioncomment-9359600) with Pierre, there is a discrepancy with observation.
>On this topic, you may have seen all the recent evidence for planets and moons having more internal heat than can be explained.

![shut_up](https://github.com/a-cosmology-group/acg/assets/135433222/131488ca-6c19-4257-9646-5d6379472ce4)
The (improved) Hubble luminosity is, whith $H_0=$ the Hubble constant:

$$
L_H = \frac{GM^2}{2R}H_0
$$

For the Earth we have the following numbers
```
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H_0 := 73.4; # Riess
H_0 := H_0*1000/Mpc;
M := 5.972*10^24;
R := 6.3781*10^6;
L_H := G*M^2/(2*R)*H_0;
                                             15
                       L_H := 0.4438693952 10
```
Continuing with the [Stefan-Boltzmann law](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law#Temperature_of_stars):
```
sigma := 5.670374419*10^(-8);
Theta := evalf((L_H/(4*Pi*R^2*sigma))^(1/4));

                         Theta := 62.55510221
```
for the excess temperature of the earth: 63 K.

### Edit

Same calculation for the sun
```
M := 1.989*10^30;
R := 696265000;
L_H := G*M^2/(2*R)*H_0;
sigma := 5.670374419*10^(-8);
Theta := evalf((L_H/(4*Pi*R^2*sigma))^(1/4));

                         Theta := 1068.951005
```
That excess temperature of 1069 K is a sensible part of the sun's temperature at the surface (5800 K) but it cannot explain for the corona.

**Reply by ExpEarth** [reply]
@HanDeBruijn 

Thanks for your input, Han.

`What you call gravitational potential energy should be the same as our self energy. But your formula is missing a factor of 2 in the denominator.`

I will check that out in _Relativity Reexamined_ .  The factor of 2 in the denominator I assume applies to relativistic objects like black holes.  In those situations it could actually help my model.

For the Earth, it is generally expressed as _U = 0.6GM^2/R_ , which is not far from your value actually.  Your calculation verifies that the energy the Earth would radiate away is rather small and its effective surface temperature would be sub-freezing if that was the only source of heat.  The earth's surface temperature is dominated by the Sun's radiation.  To test the model, we actually need to look at objects that are far away from the Sun and where other sources of heat can be eliminated.  These might include icy moons or planets, provided that tidal forces are absent.  For the Earth, expansion would be a possible _sink_ for this energy.  Most of the energy released would go into increasing the Earth's radius and thus its gravitational potential energy.

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>The factor of 2 in the denominator I assume applies to relativistic objects like black holes.

No Matt, that factor 2 applies to purely Newtonian objects. Please read the suggested section in the book.

The more I think about your _Hubble luminosity_, the more I find that it's true, but it also appears more and more trivial.
The reason is that (not only) in the **V**ariable **M**ass **T**heory the Hubble parameter $H$ is related to **Age**.
Most of the time, the "age of the universe" is put equal to $\approx 1/H$, but in the VMT it can be e.g. the Age of the Earth as well.
Fortunately, due to the fourth power law, the exact choice of $A=1/H$ has not much influence on the outcomes for excess temperatures. Obtained results may therefore be expected to remain reasonably accurate.
Whatever, quite in general we have

$$
\frac{dU}{dt} = L_H = -U.H = -U/A \quad \Longrightarrow \quad U(t) = U_0.e^{-t/A}
$$

which is precisely the well-known formula for an [Exponential decay](https://en.wikipedia.org/wiki/Exponential_decay), where $A=$ [mean lifetime](https://en.wikipedia.org/wiki/Exponential_decay#Mean_lifetime).
Maybe you can add _Hubble luminosity_ to the list of phenomena in _Natural sciences_ at this Wikipedia page :-)

**Reply by ExpEarth** [reply]
@HanDeBruijn 

While I'm happy that you are starting to like the Hubble luminosity, I am not happy that you constantly invoke these exponentially changing processes.  All the evidence suggests that the universe carries on in a more or less constant and unchanging manner.  The "age of the universe", 1/_H_ , is not really an age but the period of the cosmic cycle.  All the processes of creation and destruction are governed by this cycle.  Some examples:

**Star formation rates, SFR**
SFR/unit volume = _n H_ where _n_ is the average stellar number density

**Star death rates, SDR**
SDR/unit volume = _n H_

**rate of hydrogen fusion**
_d rho H(atomic) / dt_ = - _rho H(atomic)_ _H_

This is why we don't generally see stars older than _1/H_ .  I wrote a small paper on this in 1998.  [http://redshift.vif.com/JournalFiles/Pre2001/V05NO3PDF/v05n3EDW.pdf](url) It needs updating.  The numbers still hold true and maybe even more so.  I'd be interested to hear suggestions on what other rates could be looked at to check this.  My general impression is that people here only look at the Hubble constant in relation to the decay of light.  It's really much bigger than that.

**Reply by sahil5d** [reply]
I agree that 1/H is not the "age of the universe".

I disagree that it is exactly "the period of the cosmic cycle", because there are cosmic cycles of many kinds and their periods might be k * 1/H, for multiple k (k probably greater than 1 and less than 100 (a guess)).

> we don't generally see stars older than 1/H

I recall there were many papers (of the 1990s) that identified stars ~ 15 Gy and older in the Milky Way, but then their authors were coerced by Big Bang cosmologists to "revise" those ages to under 13.8 Gy.

**Reply by mikehelland** [reply]
What is a cosmic cycle?

**Reply by HanDeBruijn** [reply]
@ExpEarth ..
>The "age of the universe", 1/H , is not really an age but the period of the cosmic cycle.

I think that the cosmos is **not cyclic**. And proper understanding what I've written reveals that $1/H$ indeed _is not really an age_.
I invite you to figure out what really the meaning is of $A$ in my formulation of an exponential decay.
Your last example, named **rate of hydrogen fusion** contains a differential equation. And the solution is again .. a decay!

$$
\frac{d\rho_{H(atomic)}}{dt} = -\rho_{H(atomic)}H \quad \Longrightarrow \quad \rho_{H(atomic)}(t) = \rho_{H(atomic)}(0) \mbox{ } e^{-Ht}
$$

>I am not happy that you constantly invoke these exponentially changing processes.

That may be so Matt, but I am not the one to be blamed. Mathematical truth has no mercy, not for me too.

>people here only look at the Hubble constant in relation to the decay of light.

But I am not a Tired Light supporter, you know.

@sahil5d .
>there are cosmic cycles of many kinds and their periods might be k * 1/H, for multiple k

See comment and replies in [LCDM cosmological coincidences](https://github.com/orgs/a-cosmology-group/discussions/198). Quote from a [relevant reply](https://github.com/orgs/a-cosmology-group/discussions/198#discussioncomment-8311099):
>Also take notice of the fact that I'm no longer saying that $H=1/A$ or $H=2/A$ but instead that the Hubble parameter is inversely proportional to Age, with an arbitrary factor $n$, not even necessarily an integer: $H=n/A$.

Or $A=n * 1/H$ which is almost the same as what you are saying.

@mikehelland .
>What is a cosmic cycle?

Good question !

**Reply by ExpEarth** [reply]
@sahil5d 

Yes, there can be other cycles, but the one that would be the governor in the observable universe has the Hubble constant _H_ as the principal cycle parameter and 1/_H_ as the cycle period.  In my work, I find that black holes (or equivalent term) each have their own Hubble 'constant'.  Their periods are much shorter.  Can you think of some other cycles that we can look at for comparison?

`I recall there were many papers (of the 1990s) that identified stars ~ 15 Gy and older in the Milky Way, but then their authors were coerced by Big Bang cosmologists to "revise" those ages to under 13.8 Gy.`

Yes, there are some stars with greater ages but many too that have lesser ages.  The Hubble cycle would refer to the average age of stars per stellar unit mass.

@mikehelland 

The easiest way to think of it is that the universe is a sort of forest.  The trees (stars) have an average age and average lifetime.  These numbers applying to individual trees are not what is governing the cycle though.  Imagine you had the whole forest under a glass cover so gases cannot get in or go out.  It's the metabolic rate at which the trees consume and emit gases which actually controls the cycle rate.  In a steady state situation, it is the rate at which trees absorb CO2, relative to the total quantity of CO2 present, which would fix the cycle parameter.

**Reply by ExpEarth** [reply]
@HanDeBruijn 

`I think that the cosmos is not cyclic. .... Your last example, named rate of hydrogen fusion contains a differential equation. And the solution is again .. a decay!`

It's the same situation as we discussed before regarding the Hubble luminosity.  There is no exponential decay!  For every H atom that is fused into He, there is an H atom being formed somewhere else, most likely in supermassive black holes (AGNs/quasars).

`But I am not a Tired Light supporter, you know.`

This could be the root of all your difficulties!

**Reply by HanDeBruijn** [reply]
@ExpEarth . There is **no discussion** about the solution of a differential equation, Matt. Period. And _supermassive black holes_ are not a panacea for all inconsistencies that pop into people's heads.
Don't get me wrong: I understand the grand idea and might even agree with it. What is IMO lacking is a mathematical formulation that is sufficiently accurate.

**Reply by mikehelland** [reply]
The Hagedorn temperature has been tested in labs, and it does create quark-gluon plasma from heavy ions:

https://en.wikipedia.org/wiki/Hagedorn_temperature

Quasars have been measured to exceed this temperature, so it seems pretty certain quasars can boil any element into something that must reconstitute as hydrogen.

**Reply by HanDeBruijn** [reply]
@mikehelland . Quote from the Wikipedia page: _This [quark matter](https://en.wikipedia.org/wiki/QCD_matter) **may** have been detected_ [ .. ] (bold font by me).
And there is a relevant [reply in thread 200](https://github.com/orgs/a-cosmology-group/discussions/200#discussioncomment-9205176) .

**Reply by ExpEarth** [reply]
@HanDeBruijn 

`There is no discussion about the solution of a differential equation, Matt. Period. `

In this discussion of cycles, we are going in circles.  One last try.  Suppose you start off with a universe of stars and they all burn the fuel they have and then eventually die.  If you look at just those stars alone, you could write your exponential decay law for them.  But elsewhere in the universe there are new stars being formed, where that hydrogen fuel has been regenerated at the same rate as it was being consumed in stars.  So, on the cosmic scale, _there is no change_ .

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>If you look at just those stars alone, you could write your exponential decay law for them.

Ah, now we are getting somewhere! That's is precisely what I am saying. Worse, it is precisely what **you** are saying all the time too, with your graphs and your formulas. Look at his one:
![matt](https://github.com/a-cosmology-group/acg/assets/135433222/1ab03ddb-0c9b-4764-be82-3d9d3df3c2c2)
What are the labels "Earth", "Jupiter", "WD", "NS", "SMBH" for? It means: _If you look at just those_ objects _**alone**_.
If you include all the rest, then of course there is _conservation of energy_ and the equation for that is simply $dU/dt=0$ or $U=$ constant.

>So, on the cosmic scale, _there is no change_.

You are quite right. And I couldn't agree more. But your prose is different from your mathematics.

**Reply by ExpEarth** [reply]
With this graph it's a different story.  Unlike stellar fusion, we can put all the masses and mass systems in the universe on that straight line and they will emit energy at that same rate continuously, unless their radius or mass changes.  They are all giving off energy in proportion to their internal gravitational energy.  That energy in turn goes into gravity (in my model) and regenerates the CMB.  We can write _dU/dt = UH_ because we can also write _dU/dt = - UH_.  So on the whole, yes, _U_ is constant.

May I just say to ACG members generally that if there is anyone interested in doing further research on the Hubble luminosity, I am looking for coauthors on this.  I am not good at statistical analysis, as perhaps you already detected.  I consider this _the_ most important topic in astrophysics and cosmology and yet it has barely been touched!  There would be a ton of new data now.  Anyone interested can contact me at:  matt.edwards@utoronto.ca

**Reply by mikehelland** [reply]
I'm struggling a bit to understand what "Hubble luminosity" means. When I google it, I see mostly references to Hubble's law, and your papers.

The luminosity of the sun is L☉, and it has that value whether we observe it from Earth, from Pluto, from Andromeda, or from 100 Gpc. An object's luminosity is what it is, regardless of the distance from which it's viewed.

Can you elaborate a bit more on what you mean by that?

**Reply by HanDeBruijn** [reply]
>We can write $dU/dt = UH$ because we can also write $dU/dt = - UH$. So on the whole, yes, $U$ is constant.

With $dU/dt = UH$ and $U=Mc^2$ this could mean that Matt's _Hubble Luminosity_ is the (missing) counterpart of (my version of) the _Variable Mass Theory_. There is a section [Hubble Parameter](http://www.alternatievewiskunde.nl/cosmology/Flandern.htm) at my website for the math. As I've said, we're getting somewhere ..

**Reply by ExpEarth** [reply]
Sure.  The 'Hubble luminosity' term was perhaps not wisely chosen, as it leads to Google search results like you found.  Other names are welcome.

It is the simplest thing you can imagine.  Just take a body's internal gravitational potential energy (eg, for the Earth, _U_ = - 0.6 _GM^2/R)_ and multiply that by the Hubble constant, or _L = -UH_ .  This simple relationship can explain the luminosities of diverse classes of objects, as can be seen in the graph a few posts earlier.  It does not explain stellar luminosities well, as fusion processes release much more energy.  It only works well when other sources and sinks of internal heating (fusion, tidal, radioactivity) are negligible.  For the Earth, I have suggested that a lot of this energy goes into tectonic processes rather than emitted radiation.  On the cosmic scale, I link a reverse process of energy absorption to the Hubble redshift and gravity

**Reply by mikehelland** [reply]
If you were to replace H with any other value, say 1 inverse second, wouldn't a similar line appear, just with a different slope?

**Reply by ExpEarth** [reply]
You would change the value for the Hubble luminosity that way, but it would no longer be equal to the bolometric luminosity of these objects.  The point of the graph is to show they are within an order of magnitude of each other over a huge rate of values.

**Reply by HanDeBruijn** [reply]
We shall continue our calculations the way we did for the Hubble Luminosity of the Earth. But our goal is a lot more ambitious now: to calculate for the whole Observable universe. All according to (our version of) the Variable Mass Theory (VMT) of course.
[Previous calculations](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) can be summarized as $U_G + U_{CMB} = 0$, with the [Stefan-Boltzmann law](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law) for the radiation energy density of the CMB.

$$
U_G = - \frac{G M_O^2}{2R_O}.W.\mbox{VMT} .e^{-W}\quad ; \quad U_{CMB} = 4\pi.wW \left(\frac{c}{H}\right)^3 \frac{4\sigma}{c}\Theta^4
$$

Mass and radius of the Observable universe are given by

$$
M_O = \frac{W}{2}\frac{c^3}{G.H} \quad \mbox{;} \quad R_O = W\frac{c}{H}
$$

Gravitational constant $G$, lightspeed $c$, Hubble parameter $H$ are recognized. Other constants ( w , W , VMT ) are purely mathematical.

Another approach - the one employed by @ExpEarth, albeit with a @HanDeBruijn twist - is different from what I've done so far and can be summarized as $\mbox{ } 4\pi R_O^2.\sigma\Theta^4 + U_G.e^{-W}.H = 0$ . In words: [ CMB power times area + Hubble luminosity of redshifted gravitational self-energy = 0 ] at the rim of the observable universe. A calculation shall further clarify the meaning of the maths.
```
W := 3.383634283; # pure math
w := (2-exp(-W)*(W^2+2*W+2))/W; # LATER
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H := 70.5; # Mike Helland value
H := H*1000/Mpc;
c := 299792458;
VMT := int(exp(-x)/x^2,x=W..infinity);
M_O := W/2*c^3/(G*H);
R_O := W*c/H;
U_G := G*M_O^2/(2*R_O)*W*VMT; # self-energy
U_G := U_G*exp(-W)*H; # times redshift and Hubble Luminosity
sigma := 5.670374419*10^(-8);
Theta := evalf((U_G/(4*Pi*R_O^2*sigma))^(1/4));

         Theta := 2.230197141
```
This outcome is close to the one with radiation energy density, as previously calculated.
It is possible to establish a simple pure mathematical connection between the two models:

$$
\frac{\Theta_{Han}}{\Theta_{Matt}} = \left(\frac{W}{4w}\right)^{1/4} \approx 1.214821788
$$

**Reply by ExpEarth** [reply]
This will look like Greek to most of ACG, Han.  I will try to discuss within the confines of my model.  I don't see your model clearly enough (remember, I am not mathematical like you.  I prefer words!)

I also use both methods to determine the CMB temperature in my model (see section 4 of my preprint in discussion #204 ).   Here is how I would express things simply in my model.  The observable universe arose as a black hole-like structure.  During the formation process, the gravitational energy released with the infalling matter got trapped inside the exterior 'shell'.  (I have since modified my model to have a much thicker shell).  When you do this and assume the energy ends up as CMB radiation, you get a CMB temperature of 27 K.  Now this CMB radiation is constantly being redshifted and regenerated.  The CMB redshift causes gravity.  The rate of energy loss is _u_ x _H_ .   CMB regeneration comes from the Hubble luminosity, _-U_ x _H_  .  Since _u_ is about equal to _-U_ , as you say, with our above assumption the two rates are equal.  With the Hubble luminosity coming mostly from the outer shell, the temperature of the shell is a bit higher, at 29 K.  My model thus predicts 29 K as a maximum CMB temperature.  I think you assume like everyone else here that the CMB is constant at 2.7 K.  How you got the numbers to square with 2.7 K is not clear to me.  I would need to understand your model better.

**Comment by LifeIsStrange** [comment]
https://arxiv.org/abs/2407.06280
First DESI milky way release which is slightly larger than the state of the art H3 study that is key to understand wether the Milky Way rotation curve is flat or not.
Sadly they only release the early data release that is ten time smaller than the dataset they already have collected but keep proprietary..

**Reply by mikehelland** [reply]
Your link is to a "value-added catalog". It takes time to produce those. I don't think the DESI team is hiding anything, just taking their time to produce the extra data required for a comparable data set. Hopefully we get something like SDSS's quasar catalogs from DESI. They don't simply fall out of the raw data though. It takes some effort.

**Reply by LifeIsStrange** [reply]
I disagree, it takes time to produce a paper and analysises but the times it takes is roughly independent of the sample size of the dataset, otherwise if it has taken them 1 year to produce this paper and that their next one contains tens as much data will it take them ten years?
No they have not manually inspected their 400000 stellar spectras.
My conjecture is that the team behind what is one of the most important DESI survey (the only milky way DESI survey) is not a member of the official DESI consortium for inept bureaucratic reasons.
A tentative indication of this, is that their paper is not affiliated with the official DESI arxiv account
https://arxiv.org/search/astro-ph?searchtype=author&query=DESI+Collaboration
The consequence of this conjecture is that we would otherwise be already able to rule out either MOND or DM with near certainty.

**Reply by mikehelland** [reply]
With all due respect, I think you're looking at the project as more of monolith than what it is in reality.

The "I" in DESI stands for instrument. The DESI team works on the instrument. 

What you think is important (Milky Way Survey) is not what I think is important (quasar survey). 

The people that work on the instrument should be focused on making the instrument and the data that comes directly out of it.

Other teams take that data and compliment them with more analysis (which are often model dependent), and we get these value added catalogs. It's kind of like blaming the Linux kernel developers for not updating the icons on the desktop. It's not really how that works.

Personally, I wouldn't want model dependent analysis at the "kernel" level.

**Comment by LifeIsStrange** [comment]
partial evidence for matter regeneration https://phys.org/news/2024-07-discovery-early-universe.html
stars gain vertical orbit with age
https://iopscience.iop.org/article/10.3847/1538-4357/ad61e8
https://iopscience.iop.org/article/10.3847/1538-4357/ad596e

https://journals.aps.org/pra/abstract/10.1103/PhysRevA.45.7729
https://link.springer.com/article/10.1140/epjp/s13360-024-05529-w
https://www.researchgate.net/publication/360992789_Can_a_Michelson-Morley_Experiment_Designed_with_Current_Solar_Velocity_Distinguish_between_Non-Relativistic_and_Relativistic_Theories

nice citations cf emitter theory
https://www.scilit.net/publications/cited-by-search/7266377?qa=%5B%7B%22id%22%3A%22common%22%2C%22value%22%3A%22%22%7D%2C%7B%22id%22%3A%22cited_id%22%2C%22value%22%3A%5B%227266377%22%5D%2C%22operator%22%3A%22AND%22%7D%5D&sort=newest&facet=%7B%22no-filters%22%3A%5B%22true%22%5D%7D
https://arxiv.org/abs/2304.14820
variable speed of light ???????????????????????????????????
https://journals.jps.jp/doi/10.7566/JPSCP.1.014019
https://arxiv.org/abs/1607.08043
https://www.researchgate.net/publication/300147117_Measurement_of_the_speed_of_light_from_extraterrestrial_sources
https://www.researchgate.net/publication/275320370_Is_the_Speed_of_Light_Independent_of_the_Velocity_of_the_Source?_tp=eyJjb250ZXh0Ijp7ImZpcnN0UGFnZSI6Il9kaXJlY3QiLCJwYWdlIjoicHVibGljYXRpb24iLCJwcmV2aW91c1BhZ2UiOiJwdWJsaWNhdGlvbiJ9fQ
https://www.scilit.net/publications/0018dc256b384aca1e3c8e60eda4352b
https://www.scilit.net/publications/61d4beb37814da6fccc0d12733a030b5
NEW EMITTER THEORY!!!
https://www.scilit.net/publications/034c9268b99696a4d048c64b777e2b80

https://www.scilit.net/publications/5f8ac434b957e101209444af5219e385
https://www.semanticscholar.org/paper/A-new-paradox-and-the-reconciliation-of-Lorentz-and-Guo/d3366e23364a875346390fd56da3488bae37957b
https://www.semanticscholar.org/paper/A-circular-twin-paradox-Cranor-Heider/cd9d5e063fd870fc613b32f836a16540c22caa05
https://www.semanticscholar.org/paper/547d8e81cc02d0c132966c6420bc5c80d8c04dca
https://www.semanticscholar.org/paper/f9280849f96d4b474fef6405410c0e07800efd2b
https://web.archive.org/web/20120902083928/https://webspace.utexas.edu/aam829/1/m/Relativity_files/RitzEinstein.pdf
https://www.nature.com/articles/s41586-024-07824-z
https://news.ycombinator.com/item?id=41607781

Little red dots found unlikely to be quasars:
https://arxiv.org/abs/2409.18208

Previous lense thirring tests have fake empirical constraints hence the necessity of GINGER
https://www.researchgate.net/publication/370380205_Limitations_in_Testing_the_Lense-Thirring_Effect_with_LAGEOS_and_the_Newly_Launched_Geodetic_Satellite_LARES_2
https://www.scopus.com/results/citedbyresults.uri?sort=plf-f&cite=2-s2.0-77950342083&src=s&imp=t&sid=3ba40602c8996427053c33daeb7fae39&sot=cite&sdt=a&sl=0&origin=inward&editSaveSearch=&txGid=fa96a49f7066552ef6e7e539c1858eee

anomalous moon gravity? allais?
https://phys.org/news/2024-09-lunar-gravity-hint-partially-molten.html
kuiper belt https://www.popularmechanics.com/space/solar-system/a62325415/second-kuiper-belt/

Nice competitor for next gen black hole imaging
https://arxiv.org/abs/2406.12917

key paper about dust https://arxiv.org/abs/2411.00619
https://arxiv.org/abs/2411.00495
https://arxiv.org/abs/2411.00091
gaia https://arxiv.org/abs/2411.00088
https://arxiv.org/abs/2411.00654

particle physics https://arxiv.org/abs/2406.19421

https://www.math.columbia.edu/~woit/wordpress/?p=14144
https://phys.org/news/2024-11-black-hole-early-universe-consuming.html

https://phys.org/news/2024-11-future-space-telescopes-thin-membranes.html

> the CSST will not enable the
high resolution studies like those illustrated in Fig. 3–7 here for the fainter and smallest galaxie

First time I see this crucial argument for the need of a dedicated point source UVOIR
https://arxiv.org/pdf/2410.01187

mock gravity
https://preprints.opticaopen.org/articles/preprint/Light_induced_Mock_Gravity_at_the_nanoscale/24671673/1?file=43355184


https://ui.adsabs.harvard.edu/abs/1989ApJ...346....3W/abstract

google corpuscules theory of light
https://arxiv.org/pdf/physics/0411110

photoelectric effect
also wiki aberattion page astronomy shows partial aether drag evidence.moreover gaia aberattion measurement as new probe for mond

til https://en.wikipedia.org/wiki/Magnetic_helicity
https://ui.adsabs.harvard.edu/abs/2014ARA%26A..52....1F/abstract
https://ui.adsabs.harvard.edu/abs/1992ApJ...394....7S/abstract

critiques but intriguing https://www.semanticscholar.org/paper/Astronomical-evidence-concerning-non-gravitational-Newton/667f874b2db3b65dc8f4335780ace8974b41382c
https://www.semanticscholar.org/paper/Observational-Evidences-of-the-Cosmological-of-Time-Taganov/0c84d5d4149ccdc3c4d15237a0e10d8c5bddb718

https://news.ycombinator.com/item?id=42076537
https://arxiv.org/abs/2411.06945
also the zigzag lens
https://www.nature.com/articles/s41467-024-53363-6

https://arxiv.org/abs/2003.04935
https://news.ycombinator.com/item?id=42220155

**Comment by AngelBlanco19** [comment]
@LifeIsStrange 
With deep sadness I can confirm that the only "hypothesis" that has NOT been taken into account is the Absolute Truth, established, demonstrated and recognized about 2518 years ago or so. Matter must necessarily be infinite and eternal, because any other variant inevitably leads us to necessarily postulate the existence of non-existence, which is a logical aberration that does not even deserve to be taken into account.
Indeed, the Earth, the Moon, just like the Sun and their respective finite, determined and countable Gravitational Fields, intrinsically infinite each and every one, like every celestial body with Gravity and its respective Gravitational Field, EXPANDS AND GROWS. And it will do so up to the finite, determined and countable strict limit established by the Matter.
Our observable Universe, also finite, determinate and countable, grows and expands constantly and eternally, being as it is intrinsically infinite, so the theoretical limit of its radius, depending on the direction of said measurement, will remain Constant and will depend only on the antiquity of the celestial bodies with Gravity and therefore of dimensions of their respective Gravitational Fields in said direction. It will change only the names of new celestial bodies and those that go beyond this theoretical limit of the radius of our observable Universe.
Therefore, each Observer in the Universe, intrinsically infinite, will have his own and unique, exclusive, observable Universe, of finite, determined and countable dimensions, even the direction of measurement, and therefore the maximum antiquity in the "confines" of it. Therefore, every observable Universe is an infinitesimal part of the infinite Universe.
When the light emitted by a certain celestial body before passing the theoretical limit of our observable Universe and that has already passed said theoretical limit is exhausted, we will never receive information or light from said celestial body. However, as long as this light does not run out (the last emitted one does not finish reaching a certain observer), we will continue to perceive a light from a celestial body that is already outside the limits of our observable Universe, which gives rise to the so-called microwave background "radiation”.
[https://github.com/orgs/a-cosmology-group/discussions/253](https://github.com/orgs/a-cosmology-group/discussions/253)
[https://github.com/orgs/a-cosmology-group/discussions/252](https://github.com/orgs/a-cosmology-group/discussions/252)
